src/sentry/api/endpoints/grouping_levels.py

```diff
@@ -28,12 +29,18 @@ class MissingFeature(SentryAPIException):
     message = "This project does not have the grouping tree feature."
 
 
-class NotHierarchical(SentryAPIException):
+class IssueNotHierarchical(SentryAPIException):
     status_code = status.HTTP_403_FORBIDDEN
     code = "not_hierarchical"
```

Comment:
should we update the code to "issue_not_hierarchical" ?

Reply:
Will do.